app/controllers/discourse_custom_topic_lists/lists_controller.rb

```diff
@@ -3,9 +3,51 @@
 module ::DiscourseCustomTopicLists
   class ListsController < ::ApplicationController
     requires_plugin PLUGIN_NAME
+    skip_before_action :check_xhr, only: [:lists_feed]
 
     def show
       render json: success_json
     end
+
+    def lists_feed
+      discourse_expires_in 1.minute
+
+      raise Discourse::NotFound if !SiteSetting.experimental_topics_filter
+      raise Discourse::InvalidParameters if !params[:topic_list_name]
+      topic_query_opts = { no_definitions: !SiteSetting.show_category_definitions_in_topic_lists }
+
+      list_item =
+        DiscourseCustomTopicLists::CustomTopicList.new(current_user).find_by_slug(
```

Comment:
```suggestion
        DiscourseCustomTopicLists::CustomTopicList.new(list_target_user).find_by_slug(
```

Should this be like the above?